cargo-shuttle/src/client.rs

```diff
@@ -69,6 +69,32 @@ pub(crate) async fn status(api_url: ApiUrl, api_key: &ApiKey, project: &ProjectN
     Ok(())
 }
 
+pub(crate) async fn version(api_url: ApiUrl) -> Result<String> {
+    let client = get_retry_client();
+    let mut api_url = api_url;
```

Comment:
```suggestion
pub(crate) async fn version(mut api_url: ApiUrl) -> Result<String> {
    let client = get_retry_client();
```
I sometimes get mixed up with `mut` too. Also, only think this works, but I can be wrong too :see_no_evil:

Reply:
Also, maybe we should call this function something like `service_version` to prevent it from being confused with the client version?